Makefile

```diff
@@ -1,5 +1,5 @@
 export CGO_LDFLAGS := -l bpf
-export GOFLAGS := -mod=vendor
+export GOFLAGS :=
```

Comment:
No need to keep it around, let's remove it altogether.

Reply:
Sure, that seems reasonable. Sorry I didn't get back to this, I was traveling.